docker/generic/Dockerfile.kinetic

```
@@ -62,7 +66,7 @@ RUN sudo rosdep init \
 # Install Autoware
 RUN cd && mkdir /home/$USERNAME/Autoware
 COPY --chown=autoware ./ /home/$USERNAME/Autoware/
-RUN /bin/bash -c 'source /opt/ros/kinetic/setup.bash; cd /home/$USERNAME/Autoware/ros/src; git submodule update --init --recursive; catkin_init_workspace; cd ../; rosdep install -y --from-paths /home/$USERNAME/Autoware/ros/src --ignore-src --rosdistro kinetic; ./catkin_make_release'
+RUN /bin/bash -c 'source /opt/ros/kinetic/setup.bash; cd /home/$USERNAME/Autoware/ros/src; git submodule update --init --recursive; cd ../; rosdep install -y --from-paths /home/$USERNAME/Autoware/ros/src --ignore-src --rosdistro kinetic; ./colcon_release'
```

I recommend using `sudo -E` in the script to allow the build arguments to be passed. This useful when running behind a proxy.
```suggestion
RUN /bin/bash -c 'source /opt/ros/kinetic/setup.bash; cd /home/$USERNAME/Autoware/ros/src; git submodule update --init --recursive; cd ../; sudo -E rosdep install -y --from-paths /home/$USERNAME/Autoware/ros/src --ignore-src --rosdistro kinetic; ./colcon_release'
```